elasticsearch2/src/test/resources/log4j.xml

```diff
@@ -26,10 +26,6 @@
         <level value="DEBUG" />
         <appender-ref ref="console" />
     </logger>
-    <logger name="org.vertexium.query.Query.QUERY" additivity="false">
```

Comment:
Same as above, though I apparently chose to remove the second one instead of the first in this file :-)